src/components/Section/Section.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import cx from 'classnames';
+
+type SectionProps = {
+  children: React.ReactNode;
+  bgColor?: boolean;
+  className?: string;
+  type?: 'div' | 'section';
+};
+
+export const Section = ({
+  bgColor,
```

Comment:
re naming and application of different styles (bgColor and section--has-bg-color), I think we need to abstract this more as it could be about more than simply background colour

current site uses `section--odd` which is part the way there but isn't necessarily odd either :)

`section--alternate` would be a better fit for that

I can see bgColor is boolean but its name suggests differently, so perhaps `hasAlternateStyle`?

Reply:
Yep got this - it is solely about background color at the mo though isn't it?

Style from corporate:

```css
.section--odd {
  background-color: $blue05;
}
```

Maybe best to not over-engineer it if it's only setting a bg color for the time being - happy enough to rename the prop and/or class though, but think it should be descriptive (on both counts) `hasBgColor` & `.section--has-bg-color`?

Reply:
It is solely background colour at the moment but it's also a little too close to the old html attribute bgcolor and <b> and <i> elements